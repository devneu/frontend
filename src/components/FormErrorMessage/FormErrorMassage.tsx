import React from 'react';
import './FormErrorMessage.scss';

interface FormErrorMessageProps {
  message: string;
}

const FormErrorMessage = ({ message }: FormErrorMessageProps): any => (message ? <div className="error-message">{message}</div> : '');

export default FormErrorMessage;
