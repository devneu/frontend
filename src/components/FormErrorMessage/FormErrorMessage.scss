.error-message {
  color: white;
  padding: 2px 2px;
  margin: 5px;
  font-size: 13px;
  width: 100%;
  border-radius: 5px;
  background-color: #f5222d;
  text-align: center;
}