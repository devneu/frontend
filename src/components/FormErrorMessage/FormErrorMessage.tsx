import React from 'react';
import './FormErrorMessage.scss';

interface FormErrorMessageProps {
  message: string;
}

const FormErrorMessage = ({ message }: FormErrorMessageProps): any => {
  console.log(message);
  return message ? <div className="error-message">{message}</div> : '';
};

export default FormErrorMessage;
