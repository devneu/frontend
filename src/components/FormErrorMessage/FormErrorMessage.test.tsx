import React from 'react';
import renderer from 'react-test-renderer';
import FormErrorMessage from './FormErrorMessage';

describe('FormErrorMessage', () => {
  it('Should successfully render', () => {
    const wrapper = renderer.create(<FormErrorMessage message="Message error" />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
