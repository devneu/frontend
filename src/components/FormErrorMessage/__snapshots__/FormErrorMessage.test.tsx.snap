// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormErrorMessage Should successfully render 1`] = `
<div
  className="error-message"
>
  Message error
</div>
`;
